src/git-mirror/git-mirror-task.spec.ts

```diff
@@ -324,27 +333,33 @@ describe("GitMirrorTask", () => {
 
             it("should use correct file path to packed-refs file with default source clone directory", async () => {
                 await task.removePullRequestRefs();
-                expect(pathJoinStub.calledWithExactly(".", expPackedRefsFile));
-                expect(pathResolveStub.calledWithExactly(expPackedRefsFile)).to.be.true;
+                expect(pathJoinStub.firstCall.firstArg).to.equal(".");
+                expect(pathJoinStub.firstCall.args[1]).to.equal(`${sourceRepoDirectory}/packed-refs`);
+                expect(pathResolveStub.firstCall.firstArg).to.equal(expPackedRefsFile);
             });
 
             it("should use correct file path to packed-refs file with custom source clone directory", async () => {
                 const expectedCloneDir = "bar";
                 getInputStub.withArgs(sourceRepoCloneDirInputKey, false).callsFake(() => expectedCloneDir);
                 await new GitMirrorTask().removePullRequestRefs();
 
-                expect(pathJoinStub.calledWithExactly(".", `${expectedCloneDir}/packed-refs`));
+                expect(pathJoinStub.calledWithExactly(".", `${expectedCloneDir}/packed-refs`)).to.be.true;
```

Comment:
Highly subjective, but this is part of why I do not always love the BDD style interfaces. Something about the postfixes being simultaneously semantically required but syntactically optional is a little frustrating since they can be easily missed during authorship and are not always quickly discoverable during debugging